rse-ci.Rmd

```diff
@@ -229,24 +251,25 @@ script:
 - pytest
 ```
 
-After committing and pushing to your GitHub repository, Travis will now always
-run your tests in it's clean environment. If the tests fail, Travis will update
-the status to "failed" or "error". Then you can go through the Travis log and
-start debugging what went wrong, which can sometimes be a very difficult task
-and is always different depending on the project, tests, and programming language.
-All details about the build are contained in the Travis "Job log", right below
-the build overview. Searching for the problem can sometimes be quite tedious.
-In the final exercise we'll get you to intentionally fail the Travis build to
-see how it looks.
+With this in place,
+the Travis CI log located right below the build overview
+will contain all of the information that our testing command produces when tests fail.
 
-FIXME: Do we want to add more explanation about this? I feel running unit tests are basically the biggest reason to use CI.
+FIXME: Include examples here of failing tests in Python.
```

Comment:
Do we cover how to write tests somewhere else? Then shouldn't that be where the details of failing tests go? Correct me if I am wrong, but won't the travis specific part just be something like `The command pytest exited with status 1`?